community/Sean/assets/js/externals.js

```diff
@@ -0,0 +1,2 @@
+
+export * from '../../../../../../assets/js/components/webcomponent.js';
```

Comment:
Nothing to review here, but I did laugh seeing the amount of `../` necessary to get to the webcomponent file 😆

Reply:
Yeah, hence why I dumped it into a re-export 🤣